frontend/src/pages/Graph/GraphToolbar/GraphSettings.tsx

```diff
@@ -36,6 +36,9 @@ import { KialiDispatch } from 'types/Redux';
 import { KialiCrippledFeatures } from 'types/ServerConfig';
 import { getCrippledFeatures } from 'services/Api';
 
+import { serverConfig } from '../../../config';
+import { ReactNode } from 'react';
```

Comment:
Please remove this import and replace `ReactNode` by `React.ReactNode`

Reply:
updated